client/rpmtrans.c

```diff
@@ -472,6 +472,86 @@ TDNFTransAddReInstallPkgs(
     return TDNFTransAddInstallPkgs(pTS, pTdnf, pInfo);
 }
 
+uint32_t
+TDNFFetchRemoteGPGKey(
+    PTDNF pTdnf,
+    const char* pszRepoName,
+    const char* pszUrlGPGKey,
+    char** ppszKeyLocation)
```

Comment:
The closing ')' goese in a new line with one level of indentation.

Reply:
Fixed